landing.css

```diff
@@ -129,12 +142,47 @@ nav ul li {
 
   .description__cards {
     grid-template-columns: repeat(2, 1fr);
-    margin: 0 auto;
-    padding-right: 2rem;
+    gap: 2rem;
+    margin-bottom: -8rem;
+    margin-top: 6rem;
+  }
+
+  .description__card {
+    max-height: 11rem;
   }
 
   .description__goals {
     flex-direction: column;
     padding-top: 20rem;
   }
 }
+/** 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5575170)